### PR TITLE
Patch/fix unique key prop error cag

### PR DESCRIPTION
**Pull request by cagallo**
## What Features are Being Added?

- Added new book.png for favicon(research-icon.png) and edited index.html accordingly
- Added unique key to the two JSX tags for the selectedBook view to get rid of the unique key error

## What is Being Refactored?

- N/A

## Where should the reviewer start?

- App.js, public/index.html

## How should this be manually tested?

- npm start, go to localhost:3000, check dev tools

## What are the relevant tickets?

- Fix unique key prop error

## Any Other Notes/Questions/Problems?

- None, thanks y'all, get to this when you can :). Happy sat!

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ class App extends Component {
           return [<Nav location="favorites"/>, <Favorites />]
         }} />
         <Route exact path='/:isbn/selectedBook' render={({ match }) => {
-          return [<Nav />, <BookDetails isbn={match.params.isbn} />]
+          return [<Nav key={match.params.isbn + '-nav'} />, <BookDetails isbn={match.params.isbn} key={match.params.isbn + '-book-details'} />]
```
Comment (krdurrett) [inline src/components/App.js:43]: Good catch on this key prop!

**Review by krdurrett — APPROVED** [review]
Thanks for fixing that unique key error!

File: src/components/App.js (hunk 1 above)
Comment (Fordo29) [inline src/components/App.js:43]: This was such a great learning... we need a unique key for each component

File: src/components/BookDetails.js [hunk 2]
```diff
@@ -48,9 +48,9 @@ class BookDetails extends Component {
 
   determineButton = () => {
     if(this.state.selectedBook.isFavorited === 'true') {
-      return <button className="unfavorite-button" onClick={this.handleDelete}>Remove from Favorites</button>
+      return <button className="unfavorite-button" onClick={() => this.handleDelete}>Remove from Favorites</button>
```
Comment (Fordo29) [inline src/components/BookDetails.js:51]: Nice find on Ethan

File: src/components/BookDetails.js [hunk 3]
```diff
@@ -48,9 +48,9 @@ class BookDetails extends Component {
...
     } else {
-      return <button className="favorite-button" onClick={this.handleAdd}>Add to Favorites</button>
+      return <button className="favorite-button" onClick={() => this.handleAdd}>Add to Favorites</button>
```
Comment (ectweitmann) [inline src/components/BookDetails.js:53]: Awesome! Great fix.

**Review by ectweitmann — APPROVED** [review]

**Pull request merged**